plugins/generate-llms-txt.js

```diff
@@ -83,6 +83,7 @@ export default function generateLlmsTxtPlugin(options = {}) {
 
 	return {
 		name: 'generate-llms-txt',
+		apply: 'build',
 		async buildStart() {
```

Comment:
I mentioned this in the other PR but looks like it was missed: `this.emitFile` cannot be used in Vite's dev mode and it'll emit warnings over this. The closest equivalent for us would be adding a middleware likely, as nothing else consumes the result, but in lieu of that, we should limit this to builds.